.buildkite/pipeline.yml

```diff
@@ -101,6 +101,9 @@ steps:
       queue: "${BUILDKITE_AGENT_META_DATA_QUEUE:-default}"
       distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
       hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
+    # TODO: Remove this once v0.21.0 has been released
```

Comment:
Can we have build flags or something to remove `soft_fail`? We are not supporting multiple Firecracker versions, but keeping soft_fail will mask new incompatibilities.

Alternatively we could skip some tests.

Reply:
> keeping soft_fail will mask new incompatibilities

Build flags would also do this?